src/components/pages/newsletter/issues/issues.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+import Container from 'components/shared/container';
+import Heading from 'components/shared/heading';
+import Link from 'components/shared/link';
+import CalendarIcon from 'icons/calendar.inline.svg';
+
+const items = [
+  {
+    year: '2022',
+    issues: [
+      { title: 'eCHO News #21', date: 'December 10', link: '/' },
+      { title: 'eCHO News #21', date: 'December 10', link: '/' },
+      { title: 'eCHO News #21', date: 'December 10', link: '/' },
+      { title: 'eCHO News #21', date: 'December 10', link: '/' },
+      { title: 'eCHO News #21', date: 'December 10', link: '/' },
+      { title: 'eCHO News #21', date: 'December 10', link: '/' },
+      { title: 'eCHO News #21', date: 'December 10', link: '/' },
+      { title: 'eCHO News #21', date: 'December 10', link: '/' },
+      { title: 'eCHO News #21', date: 'December 10', link: '/' },
+    ],
+  },
+  {
+    year: '2021',
+    issues: [
+      { title: 'eCHO News #21', date: 'December 10', link: '/' },
+      { title: 'eCHO News #21', date: 'December 10', link: '/' },
+    ],
+  },
+];
+
+const Issues = () => (
+  <div className="bg-white py-10 md:py-20 lg:py-28">
+    <Container className="">
+      {items.map(({ year, issues }, index) => (
+        <div className="mt-10 first:mt-0 md:mt-20 lg:mt-24" key={index}>
+          <Heading className="flat-breaks xl:flat-none" tag="h3" size="md" asHTML>
+            {year}
+          </Heading>
+          <div className="mt-8 flex flex-col flex-wrap gap-x-8 gap-y-8 xs:flex-row md:mt-10 lg:mt-12">
```

Comment:
Should we use grid instead of flex here? Since not all versions of the browser support column-gap and row-gap with flex. As I remember only in the latest version.

Reply:
@vannyle yes, you're right, switched to grid